### PR TITLE
autoload cleanup + ready for composer

### PR DESCRIPTION
**Pull request by zzanzare**
I found your project interesting and wanted to use it, but I noticed several problems
- I couldn't install it via composer
- phpunit tests weren't running out of the box
- composer autoload wasn't used (your custom autoload was messing up my include path)

This pull request will make the project ready for installation by composer (#9).
Then you can just register it in packagist.org and update the README.md - remove the "repositories" part

### CODE REVIEW

**Comment by idler** [comment]
Thank you. We will test it soon.

**Comment by fezfez** [comment]
+1

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
-/.project
-/config.ini
-/tests/config.ini
-/tests/test_data/
-/db
-/.buildpath
-/.settings
+.project
```
Comment (byo) [inline .gitignore]: Is it intentional to remove '/' at the beginning of ignored file names? It was here to explicitly state that ignored files must be inside root directory of mmp

File: README.md [hunk 2]
```diff
@@ -4,3 +4,16 @@
...
+
+To install MMP, add this to your composer.json:
+```
+    "require": {
+        "mmp/mmp": "dev-master@dev"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/zzanzare/MMP"
```
Comment (byo) [inline README.md]: Please adjust this link to URL of this repository.

File: init.php [hunk 3]
```diff
@@ -1,24 +1,3 @@
...
-    $filename = $path . DIR_SEP . $class . '.class.php';
-    if(file_exists($filename))
-    {
-      require_once $filename;
-      return true;
-    }
-  }
-}
-
+require __DIR__ . '/vendor/autoload.php';
```
Comment (byo) [inline init.php]: I don't see this file in repo, I'm not familiar with composer but I'm pretty sure it would fail without it?

Comment (zzanzare) [inline init.php]: This file is created by composer automatically when you run "composer install". It does basically the same logic as you had in init.php. So to get the project running, you now do "git clone" followed by "composer install" and everything should be ready.

**Pull request merged**